local_units/views.py

```diff
@@ -253,7 +267,9 @@ def deprecate(self, request, pk=None):
         instance = self.get_object()
         serializer = LocalUnitDeprecateSerializer(instance, data=request.data)
         serializer.is_valid(raise_exception=True)
-        serializer.save()
+        self.perform_update(serializer)
+        deprecated_reason = serializer.validated_data["deprecated_reason_overview"]
+        transaction.on_commit(lambda: send_deprecate_email(instance.id, instance.created_by_id, deprecated_reason))
```

Comment:
Let's use instance.deprecated_reason from within the task and remove it from here